docker/docker-compose.yml

```diff
@@ -52,7 +52,7 @@ services:
       - azurite
   azurite:
     # See here for tags: https://mcr.microsoft.com/en-us/product/azure-storage/azurite/tags
-    image: mcr.microsoft.com/azure-storage/azurite:3.22.0@sha256:f7d8c2250b282ef1e7179eef5473ecabfd1d9ea3ade2c2b35eea5150dfe44c90
+    image: mcr.microsoft.com/azure-storage/azurite:3.23.0@sha256:7007b86e6ab9969d75fbce5e3ecbbc588bcb895502c4bf27ae8f8d39482b0df0
```

Comment:
Let's change this to use the latest tag to avoid future updates

Reply:
I assume without the sha is where your head is?

Reply:
Yeah. I actually do this already in [healthcare-shared-components](https://github.com/microsoft/healthcare-shared-components/blob/4d22f6ba099de5019e194f2ae104581366189353/build/integration-tests.yml#L4). Does that seem reasonable?

Reply:
Updated to latest.